parser-typechecker/src/Unison/Runtime/MCode.hs

```diff
@@ -899,12 +901,12 @@ emitFunction _ grpn rec ctx (FVar v) as
       App False (Env grpn j) as
   | otherwise = emitSectionVErr v
 emitFunction rns _ _ _ (FComb r) as
-  | False -- known saturated call
-    =
-      Call False n as
-  | False -- known unsaturated call
-    =
-      Ins (Name (Env n 0) as) $ Yield (BArg1 0)
+  -- | False -- known saturated call
+  --   =
+  --     Call False n as
+  -- | False -- known unsaturated call
+  --   =
+  --     Ins (Name (Env n 0) as) $ Yield (BArg1 0)
```

Comment:
@dolio  Were these "always failing" case matches here for a reason? They're triggering 'redundant match' in GHC 9.2.3

Reply:
They were just there to remind me to some day implement that feature. I don't think it's a great loss to just remove them, or comment them.